lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -298,53 +261,29 @@ class MailboxDashBoardController extends ReloadableController {
 
   bool get isDrawerOpen => scaffoldKey.currentState?.isDrawerOpen == true;
 
-  bool isSearchActive() => searchState.value.searchStatus == SearchStatus.ACTIVE;
 
   bool isSelectionEnabled() => currentSelectMode.value == SelectMode.ACTIVE;
 
-  void enableSearch() {
-    searchState.value = searchState.value.enableSearchState();
-  }
-
-  void disableSearch() {
-    searchState.value = searchState.value.disableSearchState();
-    searchQuery = SearchQuery.initial();
-    searchInputController.clear();
-    listFilterQuickSearch.clear();
-    emailReceiveTimeType.value = null;
-    searchFocus.unfocus();
-  }
-
-  void clearTextSearch() {
-    searchQuery = SearchQuery.initial();
-    searchInputController.clear();
-    searchFocus.requestFocus();
-  }
-
-  void onChangeTextSearch(String value) {
-    searchQuery = SearchQuery(value);
-  }
-
-  void updateTextSearch(String value) {
-    searchInputController.text = value;
-  }
-
-  void searchEmail(BuildContext context, String value) {
-    searchQuery = SearchQuery(value);
-    dispatchState(Right(SearchEmailNewQuery(searchQuery ?? SearchQuery.initial())));
+  void searchEmail(BuildContext context,String value) {
```

Comment:
There should be a space character between the 2 parameters. `searchEmail(BuildContext context, String value)`